mam/api/tests/test_api.c

```diff
@@ -540,8 +540,7 @@ static void test_api_trust() {
   free(payload);
   payload = NULL;
 
-  bundle_transactions_free(&bundle);
-  bundle_transactions_new(&bundle);
+  bundle_transactions_renew(&bundle);
```

Comment:
`bundle_transactions_t` is an UT_array you can reuse it by calling `utarray_clear`.

Reply:
Let me change it into `utarray_clear` here

Reply:
I wonder, is it better to use `utarray_renew` here?

Reply:
No, the `bundle` cannot be null.